sympy/physics/tests/test_secondquant.py

```diff
@@ -138,21 +138,23 @@ def test_basic_state():
 # 2019-07-24: No method move in the whole of SymPy
 @XFAIL
 def test_move1():
-    i, j = symbols('i,j')
-    A, C = symbols('A,C', cls=Function)
-    o = A(i)*C(j)
-    # This almost works, but has a minus sign wrong
-    assert move(o, 0, 1) == KroneckerDelta(i, j) + C(j)*A(i)
+    assert False
+    #i, j = symbols('i,j')
+    #A, C = symbols('A,C', cls=Function)
+    #o = A(i)*C(j)
+    ## This almost works, but has a minus sign wrong
+    # assert move(o, 0, 1) == KroneckerDelta(i, j) + C(j)*A(i)
```

Comment:
Why are these commented out?

Reply:
There is no move function in SymPy. Unless someone can figure out what this is supposed to be testing then it should really be deleted: there's no point having an xfail test if it has no chance of ever passing.

Reply:
Maybe add a note saying this then or remove altogether.

Reply:
I've removed these two tests